recipes/fiji-ilastik/ImageJ_withIlastik

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
okay now how should we name this? the current name is nice and descriptive, but `Ilastik` is capitalized which we normally don't do, then there is also mixing of underscores and CamelCase... and it might be a bit long? 😬 . Maybe `ImageJ-ilastik`? Not sure if that looks better, but it's easier to type, maybe?